cores/esp32/esp32-hal-i2c.c

```diff
@@ -445,7 +460,25 @@ void i2cInitFix(i2c_t * i2c){
     i2c->dev->fifo_data.data = 0;
     i2cSetCmd(i2c, 1, I2C_CMD_WRITE, 1, false, false, false);
     i2cSetCmd(i2c, 2, I2C_CMD_STOP, 0, false, false, false);
+    if (i2c->dev->status_reg.bus_busy) // If this condition is true, the while loop will timeout as done will not be set
+    {
+        //log_e("Busy at initialization!");
+    }
```

Comment:
maybe leave the error uncommented? otherwise what is the point of the check? :P

Reply:
The fact that you commented means it served its point. :) I left it commented out since I wanted to highlight where the error comes into play and because of the fact most errors don't have error logging enabled in the read and write methods so was unsure when we want to show and not show.

I can certainly add it any with any others we think should be there.

Any other concerns?

Reply:
I think we can uncomment the logs, since when I wrote it there was no way to control the log level :P now there is :)